### PR TITLE
[stable/postgresql] Prefix port names with protocol to comply with Istio

### PR DESCRIPTION
**Pull request by TomBrouws**
#### Is this a new chart
No

#### What this PR does / why we need it:
Istio requires service port names to be prefixed with the protocol: https://istio.io/docs/ops/deployment/requirements/

#### Special notes for your reviewer:
None

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/postgresql/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: postgresql
-version: 7.7.1
+version: 7.7.2
```
Comment (davidkarlsen) [inline stable/postgresql/Chart.yaml]:
```suggestion
version: 8.0.0
```
as folks probably depend on the service-names.

Comment (juan131) [reply stable/postgresql/Chart.yaml]: I agree.
We also need to ensure this PR isn't including any incompatibility with previous versions, and, if that's case, we need to provide the corresponding instructions to upgrade the chart.

**Review by juan131 — CHANGES_REQUESTED** [review]

File: stable/postgresql/Chart.yaml (hunk 1 above)
Comment (TomBrouws) [reply stable/postgresql/Chart.yaml]: Thanks for looking at the PR! You're right, some people might depend on the port names outside of the dependencies in the chart itself. I'll bump the version and add a note in the docs to advise people to check dependencies on port names and change them accordingly.

**Review by juan131 — APPROVED** [review]
Thanks!! Approving your PR

**Comment by TomBrouws** [comment]
@juan131 Thanks! I think you also need to remove the do-not-merge/hold label before it can go through.

**Pull request merged**